contrib/testing/kind-fast.yaml

```diff
@@ -1,25 +1,25 @@
 extraVolumes:
-  - name: cilium-dbg-binary
```

Comment:
nit: this is good cleanup, but technically should be a different commit

Reply:
agreed, i didn't initially intend to reformat the whole file but my editor got excited. Intention was just to add the needed volume and mount